src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1209,6 +1209,80 @@ is_element_type_primitive (MonoType *vector_type)
 	}
 }
 
+static MonoInst*
+emit_msb_vector_mask (MonoCompile *cfg, MonoClass *arg_class, MonoTypeEnum arg_type)
+{
+	guint64 msb_mask_value[2];
+
+	switch (arg_type) {
+		case MONO_TYPE_I1:
+		case MONO_TYPE_U1:
+			msb_mask_value[0] = 0x8080808080808080;
```

Comment:
These constants are nice and repetitive, we could optimize loading them using `dup` (later).

Reply:
Thanks, added it to the list in the PR description.